doc/man3/SSL_CTX_add1_chain_cert.pod

```diff
@@ -52,7 +52,7 @@ SSL_CTX_clear_chain_certs() clears any existing chain associated with the
 current certificate of B<ctx>.  (This is implemented by calling
 SSL_CTX_set0_chain() with B<sk> set to B<NULL>).
 
-SSL_CTX_build_cert_chain() builds the certificate chain for B<ctx> normally
+SSL_CTX_build_cert_chain() builds the certificate chain for B<ctx>. Normally
```

Comment:
FWIW, in manpages, each sentence should start on a new line.  A little known, but handy "mandoc" nit...

Reply:
Yes, I learned that a couple of months back.
For the sake of easy reviewing I typically refrain from doing such changes, 
unless I touch the given sentence anyway, but here I failed to do so. Done now.